src/common/options/rgw.yaml.in

```diff
@@ -2435,6 +2435,20 @@ options:
   services:
   - rgw
   with_legacy: true
+- name: rgw_sync_work_period
```

Comment:
is this work period still a necessary part of the design? quoting myself from above:

> then each per-shard sync process only needs to poll for the lock on log shards where it has the highest bid. once a lock is acquired, sync no longer has to drop it after rgw_sync_work_period to give other clients a chance to acquire it. we'll only stop syncing and drop our lock if we receive a notification with a higher bid

Reply:
you are correct it is possible to keep send the release lock and change the condition from time based (by work end) to by higher bid. (I think that the release_lock is still necessary).

Reply:
this config variable is no longer used and can be removed